lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1814,7 +1814,7 @@ namespace Js
             // Free unused bytes
             Assert(cbNeeded + 1 <= cbUtf8Buffer);
             *ppSourceInfo = Utf8SourceInfo::New(this, utf8Script, (int)length,
-                cbNeeded, pSrcInfo, isLibraryCode, scriptSource);
+                cbNeeded, pSrcInfo, isLibraryCode, nullptr);
```

Comment:
Thanks @digitalinfinity , i didn't realize that it has default parameter. No need to pass `nullptr` here. I will update the PR.